#109 reviewer pages

jessabean opened the pull request:
This PR addresses card 109, update reviewer pages to look similar to speaker pages.

### Changes:
- Move ratings form underneath proposal content
- Display ratings guide alongside the form instead of in a tooltip
- Display last updated meta text above proposal title

### Notes/comments:
- Would be nice to have tags not be in a select box (for both speaker and reviewers). When multiple tags are selected, the select box becomes really awkward. Maybe just a list of checkboxes?

### Screenshots:

**Reviewer view:**

![103-1](https://cloud.githubusercontent.com/assets/522911/17254778/39a57530-5584-11e6-8673-86413f4414c9.png)

**Speaker view:**

![103-2](https://cloud.githubusercontent.com/assets/522911/17254709/f494312a-5583-11e6-8769-a46235fcce7c.png)

@mghaught @gitcow

mghaught commented:
Very nice.
1. We need to keep ratings and internal comments up top as reviewers need to be able to see that without scroll especially as they revisit the page after a comment has been made.
2. Ratings guide should go back into a tooltip since it's not used all the time.  Once someone knows the guide they don't need to see that anymore.
3. I agree with you about the tag box.  I would love another less clunky way of doing this.  I have that as an enhancement card that can be tackled later.

jessabean commented:
@mghaught I've reverted back to the 3-column layout, and added a blue background to the ratings column to distinguish it from the public comments.

I've also taken a stab at addressing your suggestion to consolidate speaker, format, and track info in one place to make it easier for reviewers. 

New screenshot:

![109-4](https://cloud.githubusercontent.com/assets/522911/17277954/83054800-571d-11e6-8080-a5eac388fe8f.png)

I think it could still be worth exploring the idea of moving the public comments underneath the proposal contents, so the page isn't as squished.

mghaught commented:
Very nice Jess!  Much nicer.  I'll discuss moving public comments below the talk content with the program team to get their thoughts but I'm leaning that way.
